src/scripts/main.js

```diff
@@ -21,7 +21,25 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  if (typeof data === 'object' || data === null) {
+    return;
+  }
+
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    if (data.hasOwnProperty(key)) {
+      const li = document.createElement('li');
+
+      li.textContent = key;
+
+      if (typeof data[key] === 'object' && data[key] !== null) {
+        createTree(li, data[key]);
+      }
+
+      ul.appendChild(li);
+    }
+  }
```

Comment:
The function `createTree` does not append the `ul` element to the `element` parameter. You should add `element.appendChild(ul);` after the loop to ensure the tree is added to the DOM.